spec/02-integration/06-invalidations/03-plugins_map_invalidation_spec.lua

```diff
@@ -0,0 +1,262 @@
+local cjson        = require "cjson"
+local helpers      = require "spec.helpers"
+
+
+local POLL_INTERVAL = 0.3
+
+
+for _, strategy in helpers.each_strategy() do
+  describe("plugins map with db [#" .. strategy .. "]", function()
+
+    local admin_client_1
+    local admin_client_2
+
+    local proxy_client_1
+    local proxy_client_2
+
+    local wait_for_propagation
+
+    local service_fixture
+
+    setup(function()
+      local bp = helpers.get_db_utils(strategy, {
+        "apis",
+        "routes",
+        "services",
+        "plugins",
+        "certificates",
+      })
+
+      -- insert single fixture Service
+      service_fixture = bp.services:insert()
+
+      local db_update_propagation = strategy == "cassandra" and 0.1 or 0
+
+      assert(helpers.start_kong {
+        log_level             = "debug",
+        prefix                = "servroot1",
+        database              = strategy,
+        proxy_listen          = "0.0.0.0:8000, 0.0.0.0:8443 ssl",
+        admin_listen          = "0.0.0.0:8001",
+        db_update_frequency   = POLL_INTERVAL,
+        db_update_propagation = db_update_propagation,
+        nginx_conf            = "spec/fixtures/custom_nginx.template",
+      })
+
+      assert(helpers.start_kong {
+        log_level             = "debug",
+        prefix                = "servroot2",
+        database              = strategy,
+        proxy_listen          = "0.0.0.0:9000, 0.0.0.0:9443 ssl",
+        admin_listen          = "0.0.0.0:9001",
+        db_update_frequency   = POLL_INTERVAL,
+        db_update_propagation = db_update_propagation,
+      })
+
+      admin_client_1 = helpers.http_client("127.0.0.1", 8001)
+      admin_client_2 = helpers.http_client("127.0.0.1", 9001)
+      proxy_client_1 = helpers.http_client("127.0.0.1", 8000)
+      proxy_client_2 = helpers.http_client("127.0.0.1", 9000)
+
+      wait_for_propagation = function()
+        ngx.sleep(POLL_INTERVAL * 2 + db_update_propagation * 2)
+      end
+    end)
+
+    teardown(function()
+      helpers.stop_kong("servroot1", true)
```

Comment:
The `true` arguments here are a debug-only variable, the current policy in tests is to cleanup servroot prefixes after the tests run. Mind removing them?

Reply:
Happy to. Again as a note, this test copies the boilerplate from https://github.com/Kong/kong/blob/next/spec/02-integration/06-invalidations/02-core_entities_invalidations_spec.lua